response/core/views.py

```diff
@@ -1,9 +1,9 @@
-from rest_framework import viewsets
+from rest_framework import pagination, viewsets
 
 from response.core.models.incident import Incident
 from response.core.models.action import Action
 from response.core.models.user_external import ExternalUser
-from response.core.serializers import ExternalUserSerializer, ActionSerializer, IncidentSerializer
+from response.core import serializers
```

Comment:
It's simpler to just import the `serializers` module and qualify it - much easier to add new serializers